zeebe/gateway/src/test/java/io/camunda/zeebe/gateway/impl/configuration/GatewayCfgTest.java

```diff
@@ -174,13 +176,17 @@ public void shouldUseEnvironmentVariables() {
     setEnv("zeebe.gateway.filters.0.id", "overwrittenFilter");
     setEnv("zeebe.gateway.filters.0.className", "OverwrittenFilter");
     setEnv("zeebe.gateway.filters.0.jarPath", "./overwrittenFilter.jar");
+    setEnv("zeebe.gateway.network.soSndbuf", DEFAULT_GATEWAY_SO_SNDBUF.toString());
```

Comment:
:x: why do you set the environment to the default values?
The value provided in the env must be different than the default for the test to represent a usecase